models/attachment.go

```diff
@@ -55,15 +56,18 @@ func (a *Attachment) APIFormat() *api.Attachment {
 	}
 }
 
-// AttachmentLocalPath returns where attachment is stored in local file
-// system based on given UUID.
-func AttachmentLocalPath(uuid string) string {
-	return path.Join(setting.AttachmentPath, uuid[0:1], uuid[1:2], uuid)
+// AttachmentBasePath returns the base path of attachment in storage.
+func AttachmentBasePath(uuid string) string {
```

Comment:
I think keeping the previous method is better.
```go
func AttachmentBasePath(uuid string) string {
    if len(uuid) < 3 {
        return uuid
    }
    return path.Join(uuid[0:1], uuid[1:2], uuid)
}
```
and 
```go
func (a *Attachment) BasePath() string {
    return AttachmentBasePath(a.UUID)
}
```

Reply:
You mean keeping the method name? We only need `return path.Join(uuid[0:1], uuid[1:2], uuid)` because storage will handle the `setting.AttachmentPath` for us.

Reply:
@lunny what's exactly need to be changed? you mean the function/method name or the return value?